lerna.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: lerna
+  version: 7.3.0
+  epoch: 0
+  description: "Lerna is a fast, modern build system for managing and publishing multiple JavaScript/TypeScript packages from the same repository."
+  copyright:
+    - license: MIT
+  checks:
+    disabled:
+      - usrlocal
+  dependencies:
+    runtime:
+      - nodejs
+
+environment:
+  contents:
+    packages:
+      - bash
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - nodejs
+      - python3
+
+pipeline:
+  - name: npm install
+    runs: |
+      npm install -g ${{package.name}}@${{package.version}} -prefix ${{targets.destdir}}/usr/local/
+
+      # https://github.com/browserify/resolve/issues/288
+      sed -i 's/monorepo-symlink-test/false-positive/g' ${{targets.destdir}}/usr/local/lib/node_modules/lerna/node_modules/resolve/test/resolver/multirepo/package.json
```

Comment:
Curious what people think of this, context:
The resolve npm package, with features involving parsing `package.json` files, has a test `package.json` file that ships with it and happens to have the same name as the now-known `monorepo-symlink-test` malware npm package.

Note that the [GHSA](https://github.com/advisories/GHSA-2jcg-qqmg-46q6) is very scary, so figured since this is just a test file with a predictable path I could simply replace the name to avoid the false positive in not only our scanner but other ones too and save everyone a potential future headache.

Alternative would be an advisory, but given the severity of the GHSA I figured avoiding any flagging at all in the first place would be better?  Thoughts?

Reply:
Sounds good, will send advisory separately!